daemon/openrazer_daemon/dbus_services/dbus_methods/mamba.py

```diff
@@ -348,3 +348,54 @@ def get_supported_poll_rates(self):
     self.logger.debug("DBus call get_supported_poll_rates")
 
     return self.POLL_RATES
+
+
+@endpoint('razer.device.misc', 'setHyperPollingLED', in_sig='y')
+def set_hyperpolling_wireless_dongle_indicator_led_mode(self, mode):
+    """
+    Set the LED mode, takes in 1 char
+
+    :param mode: LED mode
+    :type mode: char
+    """
+    self.logger.debug("DBus call set_HyperPollingLED")
+
+    driver_path = self.get_driver_path('hyperpolling_wireless_dongle_indicator_led_mode')
+
+    # remember LED mode
+    self.ledmode = mode
+
+    with open(driver_path, 'w') as driver_file:
+        driver_file.write(str(mode))
+
+
+@endpoint('razer.device.misc', 'setHyperPollingPair', in_sig='s')
+def set_hyperpolling_wireless_dongle_pair(self, pid):
+    """
+    Set Pairing mode, takes in 1 string which is the PID
+
+    :param pid: product id
+    :type pid: char
+    """
+    self.logger.debug("DBus call set_HyperPollingPair")
```

Comment:
`set_HyperPollingPair` -> `set_hyperpolling_wireless_dongle_pair` (also the other two occurences please)

Reply:
Done!